lib/client-js/campusEvents.js

```diff
@@ -17,23 +17,23 @@ function getEvents() {
             var parser = new DOMParser();
             var xmlDoc = parser.parseFromString(request.responseText, "text/html");
             var events = xmlDoc.querySelectorAll('.SECalendarSummaryDate');
-            for (var i = 0; i < events.length; i++) {
-                if (events[i].className != "SECalendarSummaryDate") {
+            for (const element of events) {
```

Comment:
Unnecessary change. I would have preferred `events.forEach` if I was going to make the change.